glados.py

```diff
@@ -357,7 +363,8 @@ def process_TTS_thread(self):
                     total_samples = len(audio)
 
                     if total_samples:
-                        sd.play(audio, tts.RATE)
+                        self.play_sound(audio, tts.RATE)
+                        # sd.play(audio, tts.RATE)
```

Comment:
Please remove this line

Reply:
I assume you meant the commented line, in which case, done.  Otherwise, let me know.